pages/characters/index.tsx

```diff
@@ -9,11 +10,28 @@ import {
 	YearbookContainer,
 } from '../../app/components/characters'
 import { useResponsive } from '../../app/hooks'
-import { students } from '../../app/components/characters/utils/students'
 import { Student } from '../../app/components/characters/interfaces'
 
+const getStudents = async (query?: string) => {
+	try {
+		const res = await axios.get(
+			`${process.env.NEXT_PUBLIC_BASE_URL}/student/list?${query}`
```

Comment:
You may store it in a variable first instead of directly writing the URL as an argument